msu/systems/mod_settings/settings_panel.nut

```diff
@@ -44,6 +44,24 @@
 		}
 	}
 
+	function getAllSettings()
```

Comment:
Why not call this function `getAllSettings()` too?

Reply:
The functions have been renamed following discussion.